nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/AbstractTestTailFileScenario.java

```diff
@@ -125,7 +126,7 @@ public void testScenario(List<Action> actions) throws Exception {
 
     public void testScenario(List<Action> actions, boolean stopAfterEachTrigger) throws Exception {
         if (actions.contains(Action.ROLLOVER)) {
-            Assume.assumeTrue("Test wants to rename an open file which is not allowed on Windows", !SystemUtils.IS_OS_WINDOWS);
+            assumeTrue(!SystemUtils.IS_OS_WINDOWS, "Test wants to rename an open file which is not allowed on Windows");
```

Comment:
References to this test method should be refactored and annotated with `DisabledOnOs(OS.WINDOWS)` as opposed to using this `assumeTrue` approach.

Reply:
Going to make the change and remove the logging as well. Since the tests is only disabled if the action has `Action.Rollover`, I disabled the tests that has that action specified